github-push/handler_test.go

```diff
@@ -27,7 +27,7 @@ func Test_Handle_Push_InvalidBranch(t *testing.T) {
 		`{"ref":"refs/heads/staging"}`,
 	))
 
-	want := "refusing to build non-master branch: refs/heads/staging"
+	want := "refusing to build branch: refs/heads/staging"
```

Comment:
How about: `refusing to build target branch: x, want branch: y`

Reply:
Good idea. Fixed :+1: